mrp_subcontracting_inhibit/README.rst

```diff
@@ -0,0 +1,125 @@
+=====================================
+Inhibit subcontracting flow on demand
+=====================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fmanufacture-lightgray.png?logo=github
+    :target: https://github.com/OCA/manufacture/tree/14.0/mrp_subcontracting_inhibit
+    :alt: OCA/manufacture
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-mrp_subcontracting_inhibit
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/129/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+If a product has a bill of materials (BoM) of subcontracting type, and the
+vendor is included in the list of subcontractors of such BoM, when doing a
+purchase order for such vendor and product, it will always generate a
+subcontracting order.
+This module allows to inhibit such behavior, acting as a regular purchase, as
+this may not be convenient in some cases, as on manufacturing world, it's
+frequent that your vendor sells you directly the same product without you
+providing any raw material depending on lead times, material availability, etc.
+A check is provided at purchase order line level for easily disable the
+subcontracting part, as well as the same check at stock rule for propagating
+the preference from replenishment option.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+#. Go to *Inventory > Configuration > Settings*.
+#. Check 'Multi-Step Routes' checkbox in *Warehouse* section.
+
+Usage
+=====
+
+#. Go to *Inventory > Configuration > Routes* and edit 'Buy' rule and check 'Subcontracting inhibit'.
+#. Go to *Inventory > Products > Product* and create some product called 'Subcontract product' and Purchase tab as follows:
+#. [Vendor line 1] Vendor: Azure Interior, Subcontracting inhibited: Yes, Price: 5.
+#. [Vendor line 2] Vendor: Azure Interior, Subcontracting inhibited: No, Price: 10.
+#. Go to *Inventory > Products > Product* and create other product called 'Component'.
+#. Go to *Manufacturing > Products > Bill of materials* and create a new one with type = "Subcontracting" related to 'Subcontract product' and set 'Component' product.
```

Comment:
This two steps should go before to be able to edit the subcontracting inhibited field in the supplierinfo.

Reply:
Ok you're right, it's already changed. Thanks!